dashboard/app/models/pd/enrollment.rb

```diff
@@ -329,13 +329,13 @@ def newly_accepted_facilitator?
   # workshop id
   # @return [Integer, nil] application id or nil if cannot find any application
   def set_application_id
-    course_match = ->(application) {Pd::Application::ApplicationBase::COURSE_NAME_MAP.dig(application.try(:course).to_sym) == workshop.try(:course)}
+    course_match = ->(application) {Pd::Application::ApplicationBase::COURSE_NAME_MAP.dig(application.try(:course)&.to_sym) == workshop.try(:course)}
     pd_match = ->(application) {application.try(:pd_workshop_id) == pd_workshop_id}
 
     application_id = nil
     # Finds application from the school year of the workshop. Assumes workshops start after 6/1
     # because workshop.school_year assumes 6/1 is the start of the school year
-    Pd::Application::ApplicationBase.where(user_id: user_id, application_year: workshop.school_year).each do |application|
+    Pd::Application::ApplicationBase.where(user_id: user_id, application_year: workshop&.school_year).each do |application|
       application_id = application.id if course_match.call(application) || pd_match.call(application)
```

Comment:
Can `course_match` or `pd_match` ever be nil?

Reply:
No––they're defined above (lines 332 and 333) and return true or false. Should we be doing `&.` here as well to be super safe?

Reply:
So I suppose if we delete course_match or pd_match in that function, we're toast

Reply:
ahh right they're procs. 🤦‍♀️  Sorry, no need to do anything with them, got caught up in using `&` haha

Reply:
Yeah, makes sense. Thanks for asking!